data/example_realization_config_w_bmi_c__linux.json

```diff
@@ -49,10 +49,22 @@
                     "name": "bmi_c",
                     "params": {
                         "model_type_name": "bmi_c_cfe",
-                        "library_file": "./extern/cfe/cmake_cfe_lib/libcfemodel.so",
+                        "library_file": "./extern/alt-modular/cmake_am_libs/libcfebmi.so",
```

Comment:
I'm starting to wonder if we shouldn't take the `ld` approach to library names, that is not including the extension and let the build append the right extension.  Then we would only need a single input file, not one per OS.